client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsSchema.ts

```diff
@@ -5,35 +5,45 @@ import ClinicalDetailsFields from "models/enums/ClinicalDetailsFields";
 const requiredText = 'שדה זה הוא חובה';
 const maxText = 'תאריך ההתחלה צריך להיות מוקדם יותר מתאריך הסיום';
 const minText = 'תאריך הסיום צריך להיות מוקדם יותר מתאריך ההתחלה';
+const futureText = 'שגיאה: לא ניתן להכניס תאריך עתידי';
+const today = new Date();
```

Comment:
I think the today const should be created in the validation function so today updates in each validation.